types/setting.go

```diff
@@ -409,6 +412,19 @@ var (
 		},
 	}
 
+	SettingDefinitionBackupEngineBinaryTimeout = SettingDefinition{
+		DisplayName: "Backup Engine Binary Timeout",
```

Comment:
Can we replace the name with a better one without `Binary`?

Reply:
Sure, any suggestion?

Reply:
Modified to `BackupExecuteTimeout` for `backup-execute-timeout`. Thanks

Reply:
No for now. My thought is using a generic name.

Reply:
Modified to `BackupExecutionTimeout`: `backup-execute-timeout`,